### PR TITLE
#2583 French sentences translated into English, French-style spelling…

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
… corrected

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h [hunk 1]
```diff
@@ -50,17 +50,17 @@ class Bus: public Identifiable, public ContainedIn<VoltageLevel> {
   VoltageLevel & voltageLevel() { return parent(); }
 
 public:
-  /** tells if the tension is set */
+  /** tells if the voltage is set */
```
Comment (rosiereflo) [inline dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h:53]: This library is about to be deleted, not sure it is useful to update it... Same for others 3rd parties.
What do you think @gautierbureau ?

File: dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch [hunk 2]
```diff
@@ -2157,11 +2157,11 @@ index 000000000..d9c2e1570
 +  end matchcontinue;
 +end createDAE2Dynawo;
 +
-+//================================================================================
-+//   A.D :  Programmation en cours pour regroupement des struct => A debbuger
-+//================================================================================
++//=====================================================================================
```
Comment (rosiereflo) [inline dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch:2160]: Directly editing a patch is generally not a good idea

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch (hunk 2 above)
Comment (ErwanGuichard49) [reply dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch:2160]: These are comments, what could go wrong ?

Comment (rosiereflo) [reply dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch:2160]: windows is very sensitive to the index given after the file name ` index f6bc18ab2e..6eccb4f6ec 100644`

Comment (ErwanGuichard49) [reply dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch]: Agreed, I have restored the original version.

File: README.md [hunk 3]
```diff
@@ -371,7 +371,7 @@ Dyna&omega;o is licensed under the terms of the Mozilla Public License, v. 2.0.
...
 * [OpenModelica](https://www.openmodelica.org/), a Modelica environment developed and maintained by the Open Source Modelica Consortium distributed under a GPL V3.0 or OSMC Public License V1.2. The current version used is V1.13.2.
 * [SUNDIALS](https://computation.llnl.gov/projects/sundials), a suite of solvers developed and maintained by the Lawrence Livermore National Lab and distributed under a BSD-3-Clause license. The 4.1.0 version is currently used.
-* [SuiteSparse](http://faculty.cse.tamu.edu/davis/suitesparse.html), and in particular KLU, a LU decomposition library that is part of the suite sparse project, developed and maintained by T. A. Davis et al. at the University of Florida distributed under a LGPL-2.1+. The version 4.5.4 of suite sparse is the one used by Dyna&omega;o.
+* [SuiteSparse](http://faculty.cse.tamu.edu/davis/suitesparse.html), and in particular KLU, a LU decomposition library that is part of the suite sparse project, developed and maintained by T. A. Davis and al. at the University of Florida distributed under a LGPL-2.1+. The version 4.5.4 of suite sparse is the one used by Dyna&omega;o.
```
Comment (adriguir) [inline README.md:374]: I disagree with this change: the use of the "et al." is a standard form in scientific publications, when you don't mention the whole team contributing to a work but only the main author or contributor.

File: documentation/installation/installation.tex [hunk 4]
```diff
@@ -350,7 +350,7 @@ \section{Third parties}
...
 
-\item \href{http://faculty.cse.tamu.edu/davis/suitesparse.html} {\underline{SuiteSparse}}, a suite of sparse matrix algorithms and in particular KLU \cite{DavisKLU}, a LU decomposition library that is  developed and maintained by T. A. Davis et al. at the University of Florida. \Dynawo currently uses the V 5.4.0 version of the suite sparse library that is distributed under a LGPL-2.1+ license. KLU is used inside KINSOL and IDA to make the LU decomposition required during Newton-Raphson resolutions.
+\item \href{http://faculty.cse.tamu.edu/davis/suitesparse.html} {\underline{SuiteSparse}}, a suite of sparse matrix algorithms and in particular KLU \cite{DavisKLU}, a LU decomposition library that is  developed and maintained by T. A. Davis and al. at the University of Florida. \Dynawo currently uses the V 5.4.0 version of the suite sparse library that is distributed under a LGPL-2.1+ license. KLU is used inside KINSOL and IDA to make the LU decomposition required during Newton-Raphson resolutions.
```
Comment (adriguir) [inline documentation/installation/installation.tex:353]: Same comment as previously

File: dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h (hunk 1 above)
Comment (adriguir) [reply dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h:53]: Personally I think that 3rd parties shouldn't be corrected - they are 3rd parties that we include as they are.

**Review by adriguir — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ErwanGuichard49) [reply README.md]: You are right : "et al." is a latin expression. I have reversed this change.

File: documentation/installation/installation.tex (hunk 4 above)
Comment (ErwanGuichard49) [reply documentation/installation/installation.tex]: Reversed

File: dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h:53]: These are minor spelling and translation corrections, which do not substantially alter the third parties.

Comment (ErwanGuichard49) [reply dynawo/3rdParty/libiidm/core/include/IIDM/components/Bus.h]: I have restored the 3rdParty directory.

**Review by rosiereflo — DISMISSED** [review]

**Comment by rosiereflo** [comment]
Please let us know when you have done modifying new files so that we can review once and for all

**Comment by ErwanGuichard49** [comment]
> Please let us know when you have done modifying new files so that we can review once and for all

I have modified this PR because it has not been merged and because other spelling oddities have come to my mind. I advise the reviewers to review the PR and to merge it as soon as possible.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Photovoltaics/WECC/PVVoltageSource_INIT.mo [hunk 5]
```diff
@@ -30,8 +30,8 @@ model PVVoltageSource_INIT "Initialization model for WECC PV model with a voltag
   parameter Types.ApparentPowerModule SNom "Nominal apparent power in MVA";
   parameter Types.PerUnit RPu "Resistance of equivalent branch connection to the grid in pu (base SnRef)";
   parameter Types.PerUnit XPu "Reactance of equivalent branch connection to the grid in pu (base SnRef)";
-  parameter Types.PerUnit RSourcePu "Source resistance in per unit (typically set to zero, typical: 0..0.01)";
-  parameter Types.PerUnit XSourcePu "Source reactance in per unit (typical: 0.05..0.2)";
+  parameter Types.PerUnit RSourcePu "Source resistance in pu (base SnRef) (typically set to zero, typical: 0..0.01)";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Photovoltaics/WECC/PVVoltageSource_INIT.mo:33]: As it is a resistance, the voltage base should also be given. Same for XSourcePu. Also, have you checked it is SnRef and not SNom ?

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic [hunk 6]
```diff
@@ -52,8 +52,8 @@ SolverUnstableZMode         =             discrete events are unstable during th
 SolverUnbalanced            =             the number of algebraic/differential variables is different from the number of algebraic/differential equations in the simulated problem
 WrongLinearSolverChoice     =             the linear solver name provided is not valid
 SolverMissingParam          =             no value found for mandatory solver parameter '%1%' (parameter set '%2%', parameter file '%3%')
-SolverJacobianWithNulColumn =             jacobian has nul column %1%: equation '%2%' is ill-formed
-SolverJacobianWithNulRow    =             jacobian has nul row %1%: variable '%2%' is ill-formed
+SolverJacobianWithNulColumn =             jacobian has null column %1%: equation '%2%' is ill-formed
```
Comment (rosiereflo) [inline dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic:55]: we are speaking a column with only zero here, so let's use zero instead of null here

Comment (ErwanGuichard49) [reply dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: Corrected.

**Review by cossartque — APPROVED** [review]

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**